processing/src/main/java/org/apache/carbondata/processing/loading/DataLoadProcessBuilder.java

```diff
@@ -110,10 +113,10 @@ private AbstractDataLoadProcessorStep buildInternalForNoSort(CarbonIterator[] in
    */
   private AbstractDataLoadProcessorStep buildInternalWithNoConverter(
       CarbonIterator[] inputIterators, CarbonDataLoadConfiguration configuration,
-      SortScopeOptions.SortScope sortScope) {
+      SortScopeOptions.SortScope sortScope, boolean withoutReArrange) {
```

Comment:
can we remove all re-arrange in all flow?

Reply:
That is more change, may be refactor in another PR